src/dotnet/Chat.UI.Blazor/Pages/EmbeddedChatPage.razor

```diff
@@ -0,0 +1,114 @@
+@page "/embedded/{ChatSid}"
+@inherits ChatPage
```

Comment:
Is it not better to introduce ChatPageBase class?
Now it seems a lot of significant logic from ChatPage class is overriden: markup and ComputeState.
And it makes harder to follow what in fact is inherited and what is replaced.